source/extensions/filters/http/rbac/rbac_filter.h

```diff
@@ -66,6 +66,15 @@ class RoleBasedAccessControlFilterConfig {
            Filters::Common::RBAC::DynamicMetadataKeysSingleton::get().ShadowEngineResultField;
   }
 
+  std::string enforcedEffectivePolicyIdField() const {
+    return rules_stat_prefix_ + Filters::Common::RBAC::DynamicMetadataKeysSingleton::get()
```

Comment:
I don't really understand why are these considered stats (as they are prefixed by `rules_stat_prefix_`).
Seems like an error in the original shadow fields, an it is being persisted here as well.

I suggest considering not using a prefix at all, but will leave this to the code-owner to decide.
cc @yangminzhu @yanavlasov

Reply:
I think the prefix is useful when there are multiple RBAC filters configured.